src/modules/esl-event-listener/core/targets/wheel.target.ts

```diff
@@ -63,12 +70,17 @@ export class ESLWheelTarget extends SyntheticEventTarget {
   /** Handles wheel events */
   @bind
   protected _onWheel(event: WheelEvent): void {
+    if (this.config.skipOnScroll) this.scrollData = this.scrollData.concat(getParentScrollOffsets(event.target as Element, this.target));
```

Comment:
Cosmetic, for more convenient reading and understanding
```suggestion
    if (this.config.skipOnScroll) {
      const offsets = getParentScrollOffsets(event.target as Element, this.target);
      this.scrollData = this.scrollData.concat(offsets);
    }
```